ush/ufsda/stage.py

```diff
@@ -33,21 +33,35 @@ def gdas_fix(input_fix_dir, working_dir, config):
     # create output directories
     ufsda.disk_utils.mkdir(config['fv3jedi_fieldmetadata_dir'])
     ufsda.disk_utils.mkdir(config['fv3jedi_fix_dir'])
-    # figure out analysis resolution
-    if config['DOHYBVAR']:
-        case_anl = config['CASE_ENKF']
-    else:
-        case_anl = config['CASE_ANL']
+
+    # error checking
+    dohybvar = config['DOHYBVAR']
+    case = config['CASE']
+    case_enkf = config['CASE_ENKF']
+    case_anl = config['CASE_ANL']
+    if dohybvar and not case_enkf == case_anl:
+        raise ValueError(f"dohybvar is '{dohybvar}' but case_enkf= '{case_enkf}' does not equal case_anl= '{case_anl}'")
+
+    # set layers
     layers = int(config['LEVS'])-1
 
     # figure out staticb source
     staticb_source = config.get('STATICB_TYPE', 'gsibec')
 
-    # link staticb
-    if staticb_source in ['bump', 'gsibec']:
+    # link bump staticb
+    if staticb_source in ['bump']:
         ufsda.disk_utils.symlink(os.path.join(input_fix_dir, staticb_source, case_anl),
                                  config['fv3jedi_staticb_dir'])
 
+    # link gsibec staticb
+    if staticb_source in ['gsibec']:
+        if dohybvar:
+            case_gsibec = case_anl
+        else:
+            case_gsibec = case
```

Comment:
```
case_gsibec = case_anl if dohybvar else case
```

Reply:
Thanks.  I didn't realize the if-else structure could be collapsed into a single line.   The suggested line is succinct and readable.   

I made the change.   Subsequent testing revealed that the original logic is not correct.  `gsibec` requires that we use `case` independent of the value of `dohybvar`.   Neither the original logic nor suggested revision are necessary.   

The logical test was removed and `case` is used for all `gsibec` runs.  Change committed at 0ba33d8.